### PR TITLE
[10.0][FIX] l10n_it_reverse_charge: Check RC flag in lines of brand new vendor bill refund

### PR DESCRIPTION
**Pull request by SimoRubi**
Fix https://github.com/OCA/l10n-italy/issues/1154

--
Confermo di aver firmato il CLA https://odoo-community.org/page/cla e di aver letto le linee guida su https://odoo-community.org/page/contributing

### CODE REVIEW

File: l10n_it_reverse_charge/tests/test_rc.py [hunk 1]
```diff
@@ -408,3 +408,29 @@ def test_intra_EU_cancel_and_draft(self):
         self.assertEqual(invoice.state, 'cancel')
         invoice.action_invoice_draft()
         self.assertEqual(invoice.state, 'draft')
+
+    def test_new_refund_flag(self):
+        """Check that the lines of a new refund have the RC flag properly set.
+        """
+        supplier_extraEU = self.partner_model.create({
```
Comment (eLBati) [inline l10n_it_reverse_charge/tests/test_rc.py]: `supplier_extraEU` could be moved to `setUp`

Comment (SimoRubi) [reply l10n_it_reverse_charge/tests/test_rc.py]: @eLBati grazie della review, fatto in https://github.com/OCA/l10n-italy/pull/1155/commits/b8dbcca91fb89e842a4799bcb8d5f6d94d3ef5cf

**Comment by sergiocorato** [comment]
grazie @SimoRubi puoi unire i due commit?

**Comment by SimoRubi** [comment]
Ffatto @sergiocorato

**Pull request merged**